Elecritic/Pages/Logout.razor.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+
+using Elecritic.Database;
+using Elecritic.Helpers;
+using Elecritic.Services;
+
+using Microsoft.AspNetCore.Components;
+
+namespace Elecritic.Pages {
+    public partial class Logout {
+        [Inject]
+        private NavigationManager NavigationManager { get; set; }
+
+        [Inject]
+        private UserContext UserContext { get; set; }
+
+        [Inject]
+        public UserService UserService { get; set; }
+
+        private string ResultMessage { get; set; } = "";
+
+        public async Task LogOutUser() {
```

Comment:
Make this method synchronous because there's no need to await for anything in the body

`public void LogOutUser()`